qemu/tests/vsock_test.py

```diff
@@ -192,6 +192,7 @@ def clean(tmp_file=None):
     rec_session = send_data_from_guest_to_host(session, nc_vsock_bin,
                                                guest_cid, tmp_file)
     utils_misc.wait_for(lambda: not rec_session.is_alive(), timeout=20)
+    check_guest_nc_vsock_exit(test, session)
```

Comment:
`close_session = True`  i think.

Reply:
@yanan-fu No, I need the session later at L#197, that's why I have this variant `close_session`

Reply:
@vivianQizhu  yeap~  you are right, i missed that call.

Reply:
@vivianQizhu 
 (1/1) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.0.1.x86_64.io-github-autotest-qemu.vsock_test.negative_test: ERROR: [Errno 2] No such file or directory: '/tmp/vsock_file_D8p6nb'

Reply:
@xiangchunfu Updated, please help try again, thx.

Reply:
@vivianQizhu 
 (1/1) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.0.1.x86_64.io-github-autotest-qemu.vsock_test.negative_test: ERROR: Host connection failed